torch/nn/intrinsic/qat/modules/linear_fused.py

```diff
@@ -1,11 +1,56 @@
 import torch
 import torch.nn as nn
-import torch.nn.intrinsic as nni
 import torch.nn.functional as F
+import torch.nn.intrinsic as nni
+import torch.nn.qat as nnqat
 from torch.nn import init
 from torch.nn.parameter import Parameter
 
 
+class LinearReLU(nnqat.Linear, nni._FusedModule):
```

Comment:
For larger PRs it might be good to separate unrelated changes (such as moving the `LinearReLU` code) into a separate PR to make things easier to review.  This PR is not that complicated so it's ok here, but just an optional suggestion for future PRs.

Reply:
Ok, I'll keep that in mind